manifests/e/ESET/EndpointAntivirus/10.0.2034.0/ESET.EndpointAntivirus.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Automatically updated by the winget bot at 2022/Nov/26
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ESET.EndpointAntivirus
+PackageVersion: 10.0.2034.0
+PackageLocale: en-US
+Publisher: ESET, spol. s r.o.
+PackageName: ESET Endpoint Antivirus
+License: Copyright (c) ESET, spol. s r.o. - All rights reserved

```

```suggestion
License: Proprietary
```
Copyright is not a license.